ui/scripts/migrate-to-sql-translations.ts

```diff
@@ -0,0 +1,209 @@
+// noinspection ES6PreferShortImport
+
+/**
+ * This script will read all the nl.json files in all the repos and convert them to one big sql file for nl and en languages
+ */
+
+import * as fs from 'fs/promises';
+import { kebabCase, trim } from 'lodash';
+import * as path from 'path';
+import {
+	App,
+	Component,
+	LanguageCode,
+	TRANSLATION_SEPARATOR,
+	TranslationEntry,
+} from '../src/react-admin/modules/translations/translations.core.types';
+import { mapLimit } from 'blend-promise-utils';
+
+/**
+ * Translates dutch text using ollama llm
+ * https://gist.github.com/bertyhell/4b90cc283800150550c67abf80832aa3
+ * @param dutchText
+ */
+async function machineTranslate(dutchText: string): Promise<string> {
```

Comment:
Provide detection & dedicated error message if OLL is not running on machine.

Reply:
This is a one time script. Should we spend time on this?